apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -40,8 +41,8 @@ class PageTemplateModal extends Component {
 	);
 
 	// Parse templates blocks and memoize them.
-	getBlocksByTemplateSlugs = memoize( templates =>
-		reduce(
+	getBlocksByTemplateSlugs = memoize( templates => {
+		let rtn = reduce(
 			templates,
 			( prev, { slug, content, title } ) => {
```

Comment:
Can we name these so they are more self-explanatory? Both `rtn` and `prev` (I know the latter one was not introduced in this PR) feel like they are shortened for no reason and explaining the function call details or how reduce internals work but not really the data they hold — eg: templates, templatesBySlug or something

Reply:
Agreed. Fixed.